tasks/lint-all.js

```diff
@@ -64,6 +64,7 @@ gpii.grunt.lintAll.defaults = {
                     tables:      false,
                     code_blocks: false
                 },
+                "code-block-style": false, // Allow both fenced and indented styles
```

Comment:
The [default is "consistent"](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md046), i.e. the file needs to at least use the same convention throughout.  Can we use that?

Reply:
We discussed in chat, and agreed to allow a mixture, so that we can agree on which structure to use on a case-by-case basis.